install/kubernetes/ultimate-install.yaml

```diff
@@ -0,0 +1,905 @@
+---
+# Source: cilium/charts/agent/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cilium
+  namespace: kube-system
+---
+# Source: cilium/charts/hubble-ui/templates/serviceaccount.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  namespace: kube-system
+  name: hubble-ui
```

Comment:
nit: would you mind swapping these two so that they all are in the same order? It makes reading easier.

Reply:
done